releasenotes/notes/max-containerd-spec-5dec95b909062b4e.yaml

```diff
@@ -0,0 +1,4 @@
+---
+security:
```

Comment:
Why is this under the security section?

Reply:
Because rogue workload could consistently OOM the Agent, but it could be in `others` as well I guess.